tests/torch/test_hybrid_converter.py

```diff
@@ -127,9 +126,9 @@ def run_hybrid_llm_test(
 @pytest.mark.parametrize(
     "list_or_str_private_modules_names, expected_accuracy, has_pbs",
     [
-        ("transformer.h.0.mlp", 0.934, True),
-        (["transformer.h.0.mlp", "transformer.h.1.mlp"], 0.42, True),
-        ("transformer.h.0.mlp.c_fc", 0.986, False),
+        ("transformer.h.0.mlp", 0.95, True),
+        (["transformer.h.0.mlp", "transformer.h.1.mlp"], 0.40, True),
```

Comment:
do we have an idea why this accuracy is so low then @jfrery ?

Reply:
The "accuracy" here is the percent of similar labels in the top5 for all examples so a bit specific and I am not sure what to expect.

I think the rounding is really bad on MLP due to the outliers observed in LLMs. We might want to try smoothquant on gpt2 to see if this makes the rounding a bit less aggressive.